src/components/Footnote/stories/Footnote.intro.stories.mdx

```diff
@@ -0,0 +1,109 @@
+import { ArgsTable, Canvas, Meta } from '@storybook/addon-docs';
+
+import { FootnoteLink } from '../..';
+import { FootnoteSection } from '../..';
+
+<Meta
+  title="Components/Footnote/Intro"
+  component={FootnoteSection}
+  subcomponents={{
+    FootnoteLink,
+    'FootnoteSection.Footnote': FootnoteSection.Footnote,
+  }}
+/>
+
+# Footnote
+
+This plugin allows developers to easily embed footnotes into their Web pages, and helps to achieve WCAG 2.0 compliance when providing such footnotes
+
+Use when the end of a page's content contains a footnotes section
+
+## Example
+
+<Canvas>
+  <span>
+    <p>And here, we can see the elusive footnote in its natural habitat</p>
```

Comment:
I find this to be quiet humourous but we might need to discuss with Mena if we can leave it in